packages/i18n/src/locales/en_US.js

```diff
@@ -1578,7 +1578,120 @@ const translations = {
             description: 'Returns the specified Database of the Mongo object.'
           }
         }
-      }
+      },
+      DBRef: {
+        help: {
+          description: 'The DBRef BSON Class.',
```

Comment:
this is the Greatest Nit of all time, if there was one. The formatter in cli-repl already adds a colon after the description, so with having the dot here you end up with `The DBRef BSON Class.:`.

Reply:
Worthy!!